uwblueprint/ccbc
# Creator Model Setup

## Notion ticket link

[Creator Data Model Creation](https://www.notion.so/uwblueprintexecs/Creator-Data-Model-Creation-79e98fb7c45a42b6bb9bc1f69516ee2c)



## Implementation description
* Added `creator.model.ts` to `models` folder
* Added migration script to migrations folder to initialize table.
* Added seeding script to seed two users into creator table.



## Steps to test
1. Run Migrations First
2. Run Seeding Scripts
3. Look in PSQL table to see the two entries



## What should reviewers focus on?
* One thing to note is both creators are linked to same users as we have one user in the user table.


## Checklist
